packages/electrode-react-webapp/lib/react-webapp.js

```diff
@@ -151,22 +152,23 @@ function makeRouteHandler(routeOptions, userContent) {
     );
 
     const bundleCss = () => {
-      return WEBPACK_DEV ? devCSSBundle : cssChunk && `/js/${cssChunk.name}` || "";
+      return WEBPACK_DEV ? devCSSBundle : cssChunk && `${prodBundleBase}/js/${cssChunk.name}` || "";
```

Comment:
Good call!  Definitely will do this first thing on Monday.

(and good call on the README.  Ironically enough, I'd gone looking for something like that, but somehow didn't think of the README itself. Very grateful to you for pointing it out).